plugins/woocommerce/includes/class-wc-geolocation.php

```diff
@@ -98,6 +98,7 @@ public static function get_ip_address() {
 	 * @return string
 	 */
 	public static function get_external_ip_address() {
+		$transient_name      = null;
```

Comment:
This change can be left out.  `$external_ip_address ` can only equal false if `$transient_name` is set.

Reply:
done https://github.com/woocommerce/woocommerce/pull/41482/commits/5456d7a6208cb176ebfba3f4814f96e72726e9cf